wire-ios/WireUITests/Pages/OptionsOnSettingsPage.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+
+class OptionsOnSettingsPage: PageModel {
+
+    override var pageMainElement: XCUIElement {
+        optionsLabel
+    }
+
+    var optionsLabel: XCUIElement {
+        app.staticTexts["Options"]
+    }
+
+    var lockWithPasscodeSwitch: XCUIElement {
+        app.descendants(matching: .any)["Lock With Passcode"].firstMatch
+    }
+
+    var conversationsPageLabel: XCUIElement {
+        app.staticTexts["Conversations"]
```

Comment:
though: we need to use Constants at some point so having a target containing these ids which is accessible to the main and the tests